docs/installation/launching-ovn-kubernetes-on-kind.md

```
@@ -49,7 +49,7 @@ $ make fedora-image
 $ popd
 
 $ pushd contrib
-$ export KUBECONFIG=${HOME}/ovn.conf
+$ export KUBECONFIG=${HOME}/admin.conf
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)